src/main/java/coydir/ui/PersonListPanel.java

```diff
@@ -29,6 +29,10 @@ public PersonListPanel(ObservableList<Person> personList) {
         personListView.setCellFactory(listView -> new PersonListViewCell());
     }
 
+    public ListView<Person> getPersonListView() {
```

Comment:
Minor nit, but since this is a public method, mind adding one line of javadocs?

Reply:
noted